packages/brokeneck-fastify/lib/pluginConfig.js

```diff
@@ -4,8 +4,8 @@ const envSchema = require('env-schema')
 const S = require('fluent-json-schema')
 
 const uiSchema = S.object()
-  .prop('basename', S.string().default(''))
-  .prop('serverUrl', S.string().default('/graphql'))
+  .prop('basename', S.string())
```

Comment:
The issue comes from the empty default value.


We should choose between:

- setting that this is not an error (so we need to customize the [`strictSchema` option](https://ajv.js.org/options.html#strictschema) or the [`useDefaults`](https://ajv.js.org/options.html#usedefaults) option)
- removing the empty string as default
- setting a non-empty string as default

Reply:
@ja-bravo is this enough information?

Reply:
I tried setting a non-empty string before and it still fails with the same error. Will give the first option a try. Thanks @Eomm!